### ISSUE TITLE
Feature Engineering

### ISSUE THREAD

**Issue by Alrafikri** [agastyaharta/Data_Slinging_Slasher#4]
Apa aja nih?
@arsyac
@aafifm

**Comment by Alrafikri** [comment]
Kalau asumsiku, kan pas diskonnya di atas 10% itu si barang ngga sampai tepat waktu yak. Nah bisa jadi itu karena biaya buat shipping terpotong. Jadi aku mikir mau coba itung profit/loss.

Profit or loss = (cost of the product * profit margin) - diskon

**Comment by arsyac** [comment]
Mas Almas punya referensi yg untuk profit or loss gak mas? kalo ada aku mau bagii

**Comment by Alrafikri** [comment]
Ini mbaa @arsyac https://github.com/agastyaharta/Data_Slinging_Slasher/blob/main/STAGE_2/FeatureEngineering/Feature%20Engineering%20Profit.ipynb

**Comment by aafifm** [comment]
itu perusahaan lagi bakar duit ya 😂 rata-rata di bawah 0 profitnya.
